webapp/channels/src/components/announcement_bar/post_limits_announcement_bar/index.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import {AlertOutlineIcon} from '@mattermost/compass-icons/components';
+import type {ClientLicense} from '@mattermost/types/config';
+
+import {getUsersLimits} from 'mattermost-redux/selectors/entities/limits';
```

Comment:
This probably should be renamed, since here is used to get the posts limits, and in other component to get the user limits.

Reply:
Done.